tools/@aws-cdk/cfn2ts/lib/codegen.ts

```diff
@@ -144,15 +144,17 @@ export default class CodeGenerator {
     container: Container): Dictionary<string> {
     const propertyMap: Dictionary<string> = {};
 
+    const docs = typeDocs(resource.specName?.fqn ?? '');
+
     Object.keys(propertiesSpec).sort(propertyComparator).forEach(propName => {
       this.code.line();
       const propSpec = propertiesSpec[propName];
-      const additionalDocs = resource.specName!.relativeName(propName).fqn;
+      const additionalDocs = docs.properties[propName] || quoteCode(resource.specName!.relativeName(propName).fqn);
```

Comment:
You sure `docs.properties[propName]` doesn't require `quoteCode`? Can it hurt?

Reply:
Given that `quoteCode` looks like this:

```ts
function quoteCode(code: string): string {
  return '`' + code + '`';
}
```

I would say, no. This is intentional, we don't want to monospace the other docs.